### DISCUSSION TITLE
navbar not visible | divi builder

### DISCUSSION THREAD

**Discussion by ersingo** [bootscore/bootscore#769] [Q&A] [answered]
Hello;

I'm trying to use Divi Builder with Bootscore, but I can't show the menu, the navbar won't show up. How do I solve this. Is there a problem with me or is it something related to the plugin. I am attaching the plugin as a file.

The site I am currently working at: https://kardesler.demosite.gen.tr
[divi-builder.zip](https://github.com/bootscore/bootscore/files/15291651/divi-builder.zip)

**Comment by crftwrk** [comment, marked as answer]
Quickly inspected your page and it seems that Divi has removed entire `<header>` and `<footer>` tag. It also seems that Divi uses an own page template that doesn't fetch header and footer. Please create a subpage with normal content and check if navbar and footer is displayed as expected. Then we can investigate further.

However, installed Divi and works fine https://dev.bootscore.me/divi/?

**Comment by ersingo** [comment]
I had installed a global footer. to show it on every page. I think it blocked the system files.
